### ISSUE TITLE
Segmentation fault on minimap activation

### ISSUE THREAD

**Issue by n3tr4k** [CleverRaven/Cataclysm-DDA#28827]
**Describe the bug**  
Current UI is a bit broken and when I tried to activate minimap which disappeared I've got segfault. 

**To Reproduce**  
Try to activate minimap via keybindings (N key by default)

**Screenshots**  
![scr](https://user-images.githubusercontent.com/8440999/54640006-a83aa700-4a86-11e9-9013-df3b14d80286.png)

**Versions and configuration(please complete the following information):**
 - OS: Debian x64
 - Game Version: 0.D-711-ga159ebb18e
 - Graphics version: [Tiles or Terminal]
 - Mods loaded:
```
[
  "dda",
  "no_npc_food",
  "filthy_morale",
  "craftgp",
  "safeautodoc",
  "brightnights",
  "realguns",
  "growable-pots",
  "makeshift",
  "More_Survival_Tools",
  "alt_map_key",
  "mutant_npcs",
  "national_guard_camp",
  "more_locations",
  "FujiStruct",
  "mapgen_demo",
  "Urban_Development",
  "boats",
  "blazemod",
  "Tanks",
  "no_mutagen",
  "no_religious_Texts",
  "no_flaming_weapons",
  "more_classes_scenarios",
  "manualbionicinstall",
  "no_reviving_zombies",
  "novitamins",
  "StatsThroughSkills"
]
```

***CRASH LOG***
```
The program has crashed.
See the log file for a stack trace.
CRASH LOG FILE: ./config/crash.log
VERSION: 0.D-711-ga159ebb18e
TYPE: Signal
MESSAGE: SIGSEGV: Segmentation fault
STACK TRACE:

	./cataclysm-tiles(_Z21debug_write_backtraceRSo+0x2f) [0x5615ef93af2c]
	./cataclysm-tiles(+0x3ea34a) [0x5615ef92d34a]
	./cataclysm-tiles(+0x3ea54b) [0x5615ef92d54b]
	/lib/x86_64-linux-gnu/libc.so.6(+0x37940) [0x7f48dc3bd940]
	./cataclysm-tiles(_Z17clear_window_areaRKN10catacurses6windowE+0x13) [0x5615efe069dc]
	./cataclysm-tiles(_ZN4game20toggle_pixel_minimapEv+0x19) [0x5615ef9c2061]
	./cataclysm-tiles(_ZN4game13handle_actionEv+0x4308) [0x5615efa1ab12]
	./cataclysm-tiles(_ZN4game7do_turnEv+0x2de) [0x5615ef9e0f22]
	./cataclysm-tiles(main+0x159b) [0x5615ef80985a]
	/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xeb) [0x7f48dc3aa09b]
	./cataclysm-tiles(_start+0x2a) [0x5615ef834b6a]

	Attempting to repeat stack trace using debug symbols...
	debug_write_backtrace(std::ostream&)
	??:?
	log_crash
	crash.cpp:?
	signal_handler
	crash.cpp:?
	??
	??:0
	clear_window_area(catacurses::window const&)
	??:?
	game::toggle_pixel_minimap()
	??:?
	game::handle_action()
	??:?
	game::do_turn()
	??:?
	main
	??:?
	__libc_start_main
	/build/glibc-d2N3Ld/glibc-2.28/csu/../csu/libc-start.c:342
	_start
	??:?
```

**Comment by KorGgenT** [comment]
this is already fixed in #28823

**Comment by ZhilkinSerg** [comment]
Duplicate of #28812 (fixed in #28823).
